### DISCUSSION TITLE
Add option to color highlight messages for mod, verified etc.

### DISCUSSION THREAD

**Discussion by Thanais4** [night/betterttv#6049] [Ideas]
![image](https://user-images.githubusercontent.com/129123509/228085077-bfee3f69-f753-4fac-bfa7-55735d9be371.png)
Can we have this setting in BTTV? it is the sole reason i use FFZ.

**Comment by night** [comment]
I believe you can do this already, but for any badge.

In the BetterTTV settings, add a new `Badge` highlight and type the badge name you'd like to highlight.

`Moderator`, `VIP`, `Verified` or any other badge you see.

**Reply by Thanais4** [reply]
You are right, the menu could be a bit better i suppose. ty
